stock_sim/modules/dbWriter.py

```diff
@@ -13,30 +12,26 @@ def __init__(self):
                 'host': os.environ['DB_HOST'],
                 'database': 'stocksimdb'
                 }
-            self.stocksimdb = mysql.connector.connect(**stocksim_config)
+            self.stocksimdb = mysql.connector.connect()
```

Comment:
Thse should not be changed on dbWriter, please revert back to what was there previously

Reply:
edited